connector/mqtt/vernemq/v2k-bridge/README.md

```diff
@@ -78,18 +82,18 @@ V2K_SDK_PRODUCER_POOL_INTERVAL_MS      | 100           |
 
 #### **librdkafka**
 
-Key                                             | Default Value      |
------------------------------------------------ | ------------------ |
-V2K_KAFKA_ACKS                                  | -1                 |
-V2K_KAFKA_BATCH_NUM_MESSAGES                    | 10000              |
-V2K_KAFKA_CLIENT_ID                             | ${V2K_APP_BASEDIR} |
-V2K_KAFKA_COMPRESSION_CODEC                     | none               |
-V2K_KAFKA_DR_CB                                 | true               |
-V2K_KAFKA_ENABLE_IDEMPOTENCE                    | false              |
-V2K_KAFKA_MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION | 1000000            |
-V2K_KAFKA_METADATA_BROKER_LIST                  | kafka-server:9092  |
-V2K_KAFKA_QUEUE_BUFFERING_MAX_KBYTES            | 1048576            |
-V2K_KAFKA_QUEUE_BUFFERING_MAX_MS                | 0.5                |
-V2K_KAFKA_RETRIES                               | 2                  |
-V2K_KAFKA_RETRY_BACKOFF_MS                      | 100                |
-V2K_KAFKA_SOCKET_KEEPALIVE_ENABLE               | false              |
+Key                                             | Default Value       |
+----------------------------------------------- | ------------------- |
+V2K_KAFKA_ACKS                                  | -1                  |
+V2K_KAFKA_BATCH_NUM_MESSAGES                    | 10000               |
+V2K_KAFKA_CLIENT_ID                             | ${V2K_APP_HOSTNAME} |
+V2K_KAFKA_COMPRESSION_CODEC                     | none                |
```

Comment:
Aren't we using gzip?

Reply:
I changed the value with the env var, hadn't changed the default one. Do you think it's better?

Reply:
I think we could change the default one.